### DISCUSSION TITLE
Stream works on webbrowser, but not in radiowecker

### DISCUSSION THREAD

**Discussion by assembleer** [beabel/radiowecker#43] [Fragen & Antworten / Q&A]
Guten Tag,

Ich erhalte eine Fehlermeldung, wenn ich versuche, mehrere Streams abzuspielen.
https://27793.live.streamtheworld.com/RADIO538.mp3 zum beispiel wird über den Webbrowser abgespielt.
Wenn ich das über den Radiowecker versuche, erhalte ich eine Fehlermeldung und das Programm stürzt ab.

Hat jemand eine Idee, warum dieser Stream in diesem Projekt nicht funktioniert?

**Comment by Hotzi57** [comment]
Du kannst nur Adressen mit http:// verwenden! Aber die meisten https Seiten laufen auch mit http

**Reply by assembleer** [reply]
Vielen Dank. Das war in der Tat das Problem. Mit http funktioniert es.

**Comment by fraeggle** [comment]
Da muss ich mich mal kurz anhängen.
Versuche Energy Stuttgart  http://edge62.streamonkey.net/energy-stuttgart
Wenn ich dass eintrage hängt sich der Radiowecker komplett auf.......

Hat sich erledigt.  anderen Link verwendet und schon gehts.....
